.github/workflows/deploy-prod.yml

```diff
@@ -68,3 +68,27 @@ jobs:
           mention: here
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
+  smoke:
+    runs-on: ubuntu-latest
+    needs: backend
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node.js
+        uses: actions/setup-node@v1
+      - name: Install Dependencies
+        run: |
+          cd frontend
+          cp src/configs/dev.js src/configs/configs.js
```

Comment:
We can remove copying the config, because we're not building the app for smoke test! But if we want to keep the config, we just need to change it to use `cp src/configs/prod.js src/configs/configs.js` instead 😄 

Thank you!

Reply:
```suggestion
```